test_tableslurp.sh

```diff
@@ -2,8 +2,30 @@
 
 # Script Name: test_tableslurp.sh
 
+# Purpose: This script is called by the Makefile which passes several arguments
+# and then runs tableslurp in the background, e.g., ./tableslurp& to download a
```

Comment:
Ah, I didn't see it yesterday, but you actually run `tableslurp` in the foreground; `tableslurp` isn't a service like `tablesnap`, but a program that you run when you need it. So you could just say

```
... then runs tableslurp to download a file ...
```

Reply:
Oh right. Changed.